tests/integration/conftest.py

```diff
@@ -254,6 +255,29 @@ def datarow(dataset, image_url):
     dr.delete()
 
 
+# can be used with
+# @pytest.mark.parametrize('datarows', [<count of data rows>], indirect=True)
+# if omitted, count defaults to 1
+@pytest.fixture
+def datarows(dataset, image_url, request):
```

Comment:
Can we name this fixture `data_rows` and the one above `data_row` to stay consistent with how we use the spacing throughout the code?

Reply:
The problem was that we already had a `data_rows` function and that's why I named it `datarows`. The idea of this function is to create a flexible fixture able to create n data rows. I will try to merge both and make sure the tests work with this new version.